src/OmniSharp.Roslyn.CSharp/Services/InlayHints/InlayHintService.cs

```diff
@@ -152,6 +155,27 @@ public List<InlayHint> MapAndCacheHints(ImmutableArray<OmniSharpInlineHint> rosl
             return resultList;
         }
 
+        internal static LinePositionSpanTextChange[]? ConvertToTextChanges(TextChange? textChange, SourceText sourceText)
+        {
+            if (textChange.HasValue == false)
+            {
+                return Array.Empty<LinePositionSpanTextChange>();
+            }
+
+            var changeLinePositionSpan = sourceText.Lines.GetLinePositionSpan(textChange.Value.Span);
+            return new[]
+            {
+                new LinePositionSpanTextChange()
```

Comment:
Do we not have a helper for this conversion?

Reply:
CompletionService uses something similar `CompletionListBuilder.GetChangeForTextAndSpan()`. I will move this bit to the Helpers class and update both to use it.

Reply:
Should have looked harder, I just found the TextChangeHelper class.